src/Core/src/Core/IStepper.cs

```diff
@@ -9,6 +9,6 @@ public interface IStepper : IView, IRange
 		/// <summary>
 		/// Gets the amount by which Value is increased or decreased.
 		/// </summary>
-		double Interval { get; }
+		double Increment { get; }
```

Comment:
We can't change the name of this on the interface

It'll be too much of a breaking change

Reply:
I’ll revisit the implementation and make sure to retain the original interface name while addressing the required functionality. I’ll update the PR with a suitable fix

Reply:
Yeah, avoid braking changes always that could be possible. Also, just in case, if change something at core level in interfaces will impact other parts like test Stubs:

```
D:\a\_work\1\s\src\Core\tests\DeviceTests\Stubs\StepperStub.cs(3,47): error CS0535: 'StepperStub' does not implement interface member 'IStepper.Increment' [D:\a\_work\1\s\src\Core\tests\DeviceTests\Core.DeviceTests.csproj::TargetFramework=net9.0-ios]
D:\a\_work\1\s\src\Core\tests\DeviceTests\Stubs\StepperStub.cs(3,47): error CS0535: 'StepperStub' does not implement interface member 'IStepper.Increment' [D:\a\_work\1\s\src\Core\tests\DeviceTests\Core.DeviceTests.csproj::TargetFramework=net9.0-maccatalyst]
D:\a\_work\1\s\src\Core\tests\DeviceTests\Stubs\StepperStub.cs(3,47): error CS0535: 'StepperStub' does not implement interface member 'IStepper.Increment' [D:\a\_work\1\s\src\Core\tests\DeviceTests\Core.DeviceTests.csproj::TargetFramework=net9.0-android]
D:\a\_work\1\s\src\Core\tests\DeviceTests\Stubs\StepperStub.cs(3,47): error CS0535: 'StepperStub' does not implement interface member 'IStepper.Increment' [D:\a\_work\1\s\src\Core\tests\DeviceTests\Core.DeviceTests.csproj::TargetFramework=net9.0-windows10.0.19041.0]
C:\Users\cloudtest\.nuget\packages\microsoft.windowsappsdk\1.6.240923002\buildTransitive\Microsoft.UI.Xaml.Markup.Compiler.interop.targets(841,9): error MSB3073: The command ""C:\Users\cloudtest\.nuget\packages\microsoft.windowsappsdk\1.6.240923002\buildTransitive\..\tools\net6.0\..\net472\XamlCompiler.exe" "D:\a\_work\1\s\artifacts\obj\Core.DeviceTests\Release\net9.0-windows10.0.19041.0\win10-x64\\input.json" "D:\a\_work\1\s\artifacts\obj\Core.DeviceTests\Release\net9.0-windows10.0.19041.0\win10-x64\\output.json"" exited with code 1. [D:\a\_work\1\s\src\Core\tests\DeviceTests\Core.DeviceTests.csproj::TargetFramework=net9.0-windows10.0.19041.0]
D:\a\_work\1\s\src\Core\tests\DeviceTests\Stubs\StepperStub.cs(3,47): error CS0535: 'StepperStub' does not implement interface member 'IStepper.Increment' [D:\a\_work\1\s\src\Core\tests\DeviceTests\Core.DeviceTests.csproj::TargetFramework=net9.0-windows10.0.20348.0]
C:\Users\cloudtest\.nuget\packages\microsoft.windowsappsdk\1.6.240923002\buildTransitive\Microsoft.UI.Xaml.Markup.Compiler.interop.targets(841,9): error MSB3073: The command ""C:\Users\cloudtest\.nuget\packages\microsoft.windowsappsdk\1.6.240923002\buildTransitive\..\tools\net6.0\..\net472\XamlCompiler.exe" "D:\a\_work\1\s\artifacts\obj\Core.DeviceTests\Release\net9.0-windows10.0.20348.0\win10-x64\\input.json" "D:\a\_work\1\s\artifacts\obj\Core.DeviceTests\Release\net9.0-windows10.0.20348.0\win10-x64\\output.json"" exited with code 1. [D:\a\_work\1\s\src\Core\tests\DeviceTests\Core.DeviceTests.csproj::TargetFramework=net9.0-windows10.0.20348.0]
    9374 Warning(s)
    7 Error(s)
```

Reply:
I’ll revisit the implementation and make sure to retain the original interface name while addressing the required functionality. I’ll update the PR with a suitable fix